src/uu/date/src/date.rs

```diff
@@ -397,7 +406,7 @@ fn make_format_string(settings: &Settings) -> &str {
             Rfc3339Format::Ns => "%F %T.%f%:z",
         },
         Format::Custom(ref fmt) => fmt,
-        Format::Default => "%c",
+        Format::Default => "%a %b %-d %X %Z %Y",
```

Comment:
Layman's question: Will this not override any locales from `LC_TIME` we could use? That could be a blocker for #3143

Reply:
Yes, but this change doesn't make it any worse than then the current implementation 😄